frontend/src/store/modules/search.js

```diff
@@ -57,14 +74,21 @@ const getters = {
       Object.entries(results).map(([k, v]) => [
         k,
         (() => {
-          if (v === 0) {
-            return v;
+          if (v.results.length === 0) {
+            return { topScore: v.topScore, results: v.results };
           }
-          return v.sort((a, b) => sortResultsSearchTerm(a, b, state.searchTermString));
+          return {
+            topScore: v.topScore,
+            results: v.results
+              .sort((a, b) => sortResultsSearchTerm(a, b, state.searchTermString))
+              .slice(0, 10),
```

Comment:
This previous number was not completely random, with 6 instead of 10 you can actually see that there are other components as well in the result list + you don't have to scroll through as many items

Reply:
Sorry, I forgot to check in a local change that I mentioned in [this comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/887#discussion_r812718805).

Reply:
Cool! I'm still feel like it is looks better with fewer items though. Would be interesting to get the advisors feedback on this (or the team can look at it together on Friday perhaps)

Reply:
I agree with @inghylt that a small number is better in the quick search. As a user point of view, when using quick search, I'm interested if there is any hit and take a look at it. I don't even bother to check details of all 10 found hits. Maybe even 5 is enough.

If we are afraid that users may be confused why there are so little hits because they have missed the option for the global search, we can probably make some effort to highlight the notification of global search so that it won't be missed.

Reply:
The reasoning behind having more matches is to capture the compartmentalized metabolites, which can be more that 5.

Reply:
Summary of discussion: The chosen number is not too big of a deal but choosing either 5 or 10 seems more natural. Suggested that during demo, show advisors both 6 (5?) and 10 and let them decide what's best

Reply:
Resolved by cd3fecc210c7c76fc2a111de695b003f42e5a044 and de2a5d0af60b5f313613608bc8bd2b04a7cfda7e